package.json

```diff
@@ -151,14 +154,15 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.5.0",
     "bluebird": "3.4.6",
-    "bootstrap": "^4.1.0",
+    "bootstrap": "4.1.x",
```

Comment:
Curious if there was a strong reason for changing this? Minor version updates are pretty safe with bootstrap as they gracefully deprecate features and show warnings on compile.

Reply:
Possibly something to do with `reactstrap`?

Reply:
I updated this because I updated it and it broke the tour modal styles. It wasn't too obvious to correct.